libcloudforensics/providers/gcp/internal/common.py

```diff
@@ -85,6 +86,26 @@ def GenerateDiskName(snapshot: 'compute.GoogleComputeSnapshot',
   return disk_name
 
 
+def StampAndTruncateName(prefix: str,
```

Comment:
I'd use a more generic name e.g. `GenerateUniqueInstanceName` to separate the naming from the logic of the function. The docstring is more than enough to figure out what's going on behind the scenes.

Reply:
Done